pages/app/my-courses.js

```diff
@@ -1,14 +1,101 @@
-import { useAuthContext } from '../../utils/context/AuthProvider'
+// MyCourses widget
+// Conditionally render Student or Mentor View
+import { useEffect, useState } from 'react';
+import { Grid } from '@mui/material';
+import MyCourses from '../../components/MyCourses/MyCourses.js';
 
-export default function MyClasses() {
-  const { logout } = useAuthContext();
+const data = [
```

Comment:
Could move dummy data out of file and import, but if you're cool with it no worries